Documentation/installation/README.md

```diff
@@ -38,6 +38,12 @@ If you didn't enable Developer Mode using Xcode you will be asked to authorize t
 sudo /usr/sbin/DevToolsSecurity -enable
 ```
 
+You might also need to add your user to the developer group:
+
+```
+sudo dscl . append /Groups/_developer GroupMembership your_username
+```
```

Comment:
Can `your_username` be replaced with `$(whoami)`?

Reply:
Second this, seems cleaner from a copy/paste perspective for users.

Reply:
Done.